explorer/GrapherGrammar.ts

```diff
@@ -144,6 +144,16 @@ export const GrapherGrammar: Grammar = {
             cssClass: "",
         })),
     },
+    selectedFacetStrategy: {
+        ...EnumCellDef,
+        keyword: "selectedFacetStrategy",
+        description: "Whether the chart should be faceted or not",
+        terminalOptions: Object.keys(FacetStrategy).map((keyword) => ({
+            keyword,
+            description: "",
+            cssClass: "",
+        })),
+    },
```

Comment:
@danielgavrilov As a heads up, it's not clear to me that selecting one faceting strategy globally for an explorer is the right way to go here. It might be enough to get us started.